test/utils/image/manifest.go

```diff
@@ -292,6 +298,8 @@ func ReplaceRegistryInImageURL(imageURL string) (string, error) {
 		registryAndUser = gcrReleaseRegistry
 	case "docker.io/library":
 		registryAndUser = dockerLibraryRegistry
+	case "quay.io/ggriffiths":
```

Comment:
docker.io/ggriffiths or quay.io/ggriffiths?

Reply:
docker.io/ggriffiths - just fixed this. I had it pushed to both